tutorials/katacoda/thanos/3-receiver/step3.md

```diff
@@ -0,0 +1,54 @@
+# Configure Prometheus Remote Write
+
+Our problem in the last step was that we have have not yet configured Prometheus to `remote_write` to our `Thanos Receive` instance.
+
+We need to tell `prometheus-batcave` & `prometheus-batcomputer` where to write their data to.
+
+## Update Configuration
+
+The docs for this configuration option can be found [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+
+<pre class="file" data-filename="prometheus-batcave.yaml" data-target="replace">
+global:
+  scrape_interval: 5s
+  external_labels:
+    cluster: batcave
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['0.0.0.0:9090']
+remote_write:
+- url: 'http://0.0.0.0:10908/api/v1/receive'
+</pre>
+
+<pre class="file" data-filename="prometheus-batcomputer.yaml" data-target="replace">
+global:
+  scrape_interval: 5s
+  external_labels:
+    cluster: batcomputer
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['0.0.0.0:9090']
```

Comment:
Is it intended to scrape port 9090? For batcomputer, I think it is running on 9091?